core/src/test/scala/org/apache/spark/sql/delta/ExternalLogStoreSuite.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.sql.delta
+
+// TODO: make non-abstract and implement when tests when DynamoDBLogStore is ready
+abstract class ExternalLogStoreSuite extends PublicLogStoreSuite {
```

Comment:
why is this ExternalLogStoreSuite and not DynamoDBLogStoreSuite?

also.. why even have this class as a placeholder?

Reply:
It's better to not rush to build a public `ExternalLogStore` API in the first version. We can keep it in delta-storage-dynamodb for now.